docs/user_docs/reference/cli/kusion/_category_.json

```diff
@@ -1,4 +1,4 @@
 {
-  "label": "Kusion",
+  "label": "Kusion Command Line",
```

Comment:
It is already under the cli directory, so it may be more appropriate to rename this as "Kusion Commands"

Reply:
OK, I've updated the label to `Kusion Commands`.